cli/azd/test/functional/testdata/recordings/Test_CLI_VsServer/LiveDeployRefresh.dotnet.yaml

```diff
@@ -0,0 +1,79 @@
+version: "1.0"
```

Comment:
This is just really cool to see - I love how we are able to leverage the recorder in this way.  Thanks for adding such foundational technology to our testing strategy, @weikanglim!